pkg/sharding/apiserver/delegating_storage.go

```diff
@@ -120,10 +119,7 @@ func (s *delegatingStorage) routedRequest(ctx context.Context, resourceVersion *
 	if err != nil {
 		return nil, nil, err
 	}
-	identifier, _, err := genericcontrolplane.ParseClusterName(clusterName)
-	if err != nil {
-		return nil, nil, err
-	}
+	identifier := "fake"
```

Comment:
Is this more of a "FIXME get it compiling for now, come back to it later"?

Reply:
Yeah, I am just turning this off for now, will replace in a follow-up with #353